R/generateIIASASubmission.R

```diff
@@ -75,7 +75,8 @@ generateIIASASubmission <- function(mifs = ".", # nolint cyclocomp_linter
                                     generatePlots = FALSE,
                                     timesteps = c(seq(2005, 2060, 5), seq(2070, 2100, 10)),
                                     checkSummation = TRUE,
-                                    mappingFile = NULL) {
+                                    mappingFile = NULL,
+                                    naAction = "na.omit") {
```

Comment:
Feel a little uneasy about how I've done this - is there a better way to be passing the option for how to handle NA's into the method? Instead of passing a function as a string maybe an Enum which can be switched on to control the possible options?

Reply:
What is the alternative to "na.omit" that you want to use. Maybe that is simply a better default?

Reply:
Want to keep the NA's in the output so used 'na.pass'

Reply:
Thanks